_authors/kmfrick.md

```diff
@@ -2,9 +2,35 @@
 layout: author
 short_name: kmfrick
 name: Kevin Michael Frick
-role: Funnel Chief
+role: Funnel Engineer
 github: kmfrick
 discord: vaporK#7687
 ---
 
-This page is work in progress!
+Studio Ingegneria Informatica al [Collegio Superiore](http://www.collegio.unibo.it/it) dell'Università di Bologna. 
+Negli ultimi mesi della triennale ho lavorato come studente ricercatore al laboratorio [Robotics and Mechatronics](https://www.ram.eemcs.utwente.nl/) della University of Twente nei Paesi Bassi, nell'ambito della computer vision.
+
+Al liceo, alle olimpiadi di informatica, ho scoperto la bellezza della teoria della complessità computazionale, della teoria dei grafi, della programmazione dinamica e simili. 
+All'università mi sono interessato allo studio delle architetture dei calcolatori, che mi affascina quanto la statistica e il machine learning.
+I modelli matematici e statistici e lo sviluppo a basso livello sono quindi diventati i miei interessi principali.
+
+Nel mio tempo libero suono il basso, cucino e vado a teatro. 
+Per un periodo ho anche recitato. 
+Sono convinto che la pratica sia una forma d'arte e perciò mi impegno a lasciarmi guidare dalla ricerca di bellezza ed eleganza nel mio lavoro.
```

Comment:
Quale "pratica"? Parola sbagliata e volevi dire "programmazione"? Ho sbagliato a leggere io?

Reply:
No, mi riferisco in generale al "lavoro pratico", come lo sviluppo software. Probabilmente però non è chiaro, in effetti.

Reply:
Ho letto la descrizione in inglese dove hai scritto _practical work_. Qunidi...il lavoro pratico?

Reply:
Ha senso, lol. Correggo.

Reply:
Ok, magari cambialo in "lavoro pratico"  così si capisce meglio.